app/views/users/index.html.erb

```diff
@@ -28,6 +29,18 @@
                       <%= user.admin? ? 'Admin' : 'User' %>
                     <% end %>
                   </td>
+                  <td>
+                    <% if user.admin? %>
+                      <%= 'Admins can deactivate themselves' %>
```

Comment:
Nem hiányzik innen egy "not"? "Admins cannot be deactivated."

Reply:
így is lehet fogalmazni, de a lényeg az lenne, hogy az adminok nem tudnak más adminokat deaktiválni csak önnön magukat